### PR TITLE
Examples pass ansible-lint.

### PR DESCRIPTION
**Pull request by jeffmcutter**

### CODE REVIEW

File: inventories/inventory_loop_hosts/playbook_bad.yml [hunk 1]
```diff
@@ -1,12 +1,14 @@
-- name: provision hosts in a bad way
+- name: Provision hosts in a bad way
```
Comment (djdanielsson) [inline inventories/inventory_loop_hosts/playbook_bad.yml:1]: I think this playbook is supposed to be bad so maybe it shouldn't pass lint?

**Review by djdanielsson — CHANGES_REQUESTED** [review]
I did a quick skim and I think at least this one playbook is supposed to not be the way to do it so I am not sure if we care or want it to pass lint? Just a thought, I am not sure if there are any others like this it was the only one I noticed.

File: inventories/inventory_loop_hosts/playbook_bad.yml (hunk 1 above)
Comment (jeffmcutter) [reply inventories/inventory_loop_hosts/playbook_bad.yml:1]: I considered that as well, but was thinking the focus was more around the approach to the problem than about the formatting of the file.  I can change it back if we want.

Comment (djdanielsson) [reply inventories/inventory_loop_hosts/playbook_bad.yml:1]: @ericzolf what is your opinion here?

Comment (ericzolf) [reply inventories/inventory_loop_hosts/playbook_bad.yml:1]: I agree with Jeff, let's keep it 100% clean unless we have a very good reason (or the next person going through this exercise will trip over it).

**Review by ericzolf — APPROVED** [review]

**Review by djdanielsson — APPROVED** [review]
LGTM

**Pull request merged**